### DISCUSSION TITLE
Object identification using tagger

### DISCUSSION THREAD

**Discussion by abhika2s** [MadAnalysis/madanalysis5#209] [Q&A] [answered]
I went through the tagger section in [https://arxiv.org/pdf/2006.09387.pdf](https://arxiv.org/pdf/2006.09387.pdf). It seemed that c-tagging is possible just like b-tagging, however, the b-tagging is accompanied by a parametrization as follows:

define tagger b as b \
tanh(PT/400)*(85/(4+0.252*PT)) \
[ABSETA <= 2.5]
define tagger b as b 0 [ABSETA > 2.5]
define tagger c as b \
tanh(0.018*PT)*(1/(4+0.0052*PT))
define tagger j as b 0.01+3.8e-05*PT

which I couldn't find in the referenced paper in that section. How is this constructed? Is there such parametrization for c-tagging too?
Also if I am using the tagger function for c-tagging, should the imported file be the .hepmc file generated by Pythia or I can also work with the Basic Reco .lhe generated in MG5 (with b tagging done)?

**Comment by jackaraz** [comment, marked as answer]
Hi @abhika2s, all the simulation you see in that paper has been tuned to match the default Delphes cards. The smearing parameterization changes for every analysis; hence should be checked with respect to the analysis that you are taking as the basis. Each experimental analysis refers to specific fit parameters for tagging, which can be used in a recast.

The detector simulation in SFS is designed to work with showered and hadronised event samples not with matrix element-level events.

cheers
